README.md

```diff
@@ -101,15 +101,11 @@ When submitting a PR, please abide by our coding guidelines:
 + Code must adhere to our [coding style](https://github.com/thrasher-corp/gocryptotrader/blob/master/.github/CONTRIBUTING.md).
 + Pull requests need to be based on and opened against the `master` branch.
 
-## Compiling instructions
+## Compilation and run instructions
```

Comment:
Super-Duper Nit: "Compilation" tastes weird
``` Compiling and Running```

Reply:
Considered also splitting sections into `Compiling` and `Running`

Reply:
[done sir](https://github.com/thrasher-corp/gocryptotrader/pull/1623/commits/00ebc202a653e6722b7d9a6512a9c71bd70fd319)